SQL/New_patches/2021-07-28_diagnosis_evolution.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE `diagnosis_evolution` (
+  `DxEvolutionID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) DEFAULT NULL,
+  `ProjectID` int(10) unsigned DEFAULT NULL,
+  `visitLabel` varchar(255) DEFAULT NULL,
+  `instrumentName` varchar(255) DEFAULT NULL,
+  `sourceField` varchar(255) DEFAULT NULL,
+  `orderNumber` int(10) unsigned DEFAULT NULL,
+  PRIMARY KEY (`DxEvolutionID`),
+  UNIQUE KEY `TrajectoryName` (`Name`),
+  CONSTRAINT `FK_DxEvolution_1` FOREIGN KEY (`ProjectID`) REFERENCES `Project` (`ProjectID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `candidate_diagnosis_evolution` (
+  `ID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `CandID` int(6) NOT NULL,
+  `DxEvolutionID` int(10) unsigned NOT NULL,
+  `Diagnosis` text DEFAULT NULL,
+  `Confirmed` enum('Y', 'N') DEFAULT NULL,
+  `LastUpdate` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`ID`),
+  UNIQUE KEY `candidateDxEvolution` (`CandID`, `DxEvolutionID`),
```

Comment:
and then i think you can make the Primary Keys CandID and DxEvolution, and remove the ID key. kind of like the candidate_consent_rel table if i understand correctly that only the CandID and DxEvolutionID combination needs to be unique

Reply:
dxEvolutionID refers to the configuration id, whereas the id of this table refers to the candidate's actual diagnosis for that configuration.

more like config / configSettings table